.github/workflows/deploy.yml

```diff
@@ -197,3 +198,49 @@ jobs:
     steps:
       - name: TODO - Documentation Deploy Steps
         run: echo "TODO - Documentation Deploy Steps"
+
+      - name: Checkout Project
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3.7.0
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'yarn'
+
+      - name: Uncache Dependencies
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: node-modules-${{ hashFiles('./yarn.lock') }}
+
+      - name: Uncache Distributables
+        uses: actions/cache@v3
+        with:
+          path: '**/dist'
+          key: dist-${{ env.rid }}
+
+      - name: Synchronize Packages
+        run: yarn
+
+      - name: Build Docs
+        run: yarn run build:docs
+
+      - name: Build Samples
+        run: yarn run samples:build
+
+      - name: Provide the identity
+        run: |
+          git config --global user.email "priyakesari1993@gmail.com"
+          git config --global user.name "Priya Kesari"
```

Comment:
We shouldn't be providing our name and email to this. Please check how is this done in the master branch. We've to follow that.